providers/ibm/ibm_provider.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2019 The Terraformer Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ibm
+
+import (
+	"errors"
+
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils"
+	"github.com/GoogleCloudPlatform/terraformer/terraformutils/providerwrapper"
+)
+
+type IBMProvider struct {
+	terraformutils.Provider
+	ResourceGroup string
+}
+
+func (p *IBMProvider) Init(args []string) error {
+	if args[0] != "" {
+		p.ResourceGroup = args[0]
+	}
+	return nil
+}
+
+func (p *IBMProvider) GetName() string {
+	return "ibm"
+}
+
+func (p *IBMProvider) GetProviderData(arg ...string) map[string]interface{} {
+	return map[string]interface{}{
+		"provider": map[string]interface{}{
+			"ibm": map[string]interface{}{
+				"version": providerwrapper.GetProviderVersion(p.GetName()),
```

Comment:
not need anymore https://github.com/GoogleCloudPlatform/terraformer/blob/6794a293fcb931c538821a3a67db274fdde89343/terraformutils/terraformoutput/hcl.go#L37